ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Impl.hs

```diff
@@ -284,13 +284,13 @@ closeDBImpl
   :: forall m hash. (HasCallStack, IOLike m)
   => ImmutableDBEnv m hash
   -> m ()
-closeDBImpl ImmutableDBEnv {..} = releaseAll _dbRegistry `finally` do
+closeDBImpl ImmutableDBEnv {..} = (releaseAll _dbRegistry `finally`) $ mask_ $ do
```

Comment:
I think this `mask_` is not needed; the finalizer from `finally` will be run with exceptions masked.

Reply:
Fixed, thanks.